aptos-move/framework/aptos-experimental/sources/trading/market/market_types.move

```diff
@@ -103,8 +103,8 @@ module aptos_experimental::market_types {
         validate_bulk_order_placement_f: |address, vector<u64>, vector<u64>, vector<u64>, vector<u64>| bool has drop + copy,
         // place_maker_order_f arguments: account, order_id, is_bid, price, size, order_metadata
         place_maker_order_f: |address, OrderIdType, bool, u64, u64, M| has drop + copy,
-        // cleanup_order_f arguments: account, order_id, is_bid, remaining_size
-        cleanup_order_f: |address, OrderIdType, bool, u64| has drop + copy,
+        // cleanup_order_f arguments: account, order_id, is_bid, remaining_size, order_metadata
+        cleanup_order_f: |address, OrderIdType, bool, u64, Option<M>| has drop + copy,
```

Comment:
Since this is specific to single order which is guaranteed to have metadata, this shouldn't be an Option. Any reason you need Option here?